spec/views/spree/admin/payment_methods/index.html.haml_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+
+describe "spree/admin/payment_methods/index.html.haml" do
+  before do
+    controller.singleton_class.class_eval do
+      helper_method :new_object_url, :edit_object_url, :object_url
+
+      # Fill the helper methods below accordingly to your necessity expanding the test
+
+      def new_object_url
+        ""
+      end
+
+      def edit_object_url(object, options = {})
+        ""
+      end
+
+      def object_url(object = nil, options = {})
+        ""
+      end
```

Comment:
To make the hack look smaller I'd:
- remove the comment line # Fill...
- use the single line notation:
```
      def new_object_url() "" end
      def edit_object_url(object, options = {}) "" end
      def object_url(object = nil, options = {}) "" end
```

Reply:
I agree with the single line notation but I would stay with the comment. I think it will help new developers understand why those methods are there.

Reply:
But you understand this codebase more than me so I will be happy to remove it if you think it is not necessary (I particular don't like code with comments but I thought it would be useful on this case)

Reply:
it's fine, as you wish. it triggered me because it's about the future. no comment should be about the future, just the current state.

Reply:
Removed!